.travis.yml

```diff
@@ -1,8 +1,22 @@
 language: node_js
 sudo: false
+
 node_js:
  - stable
-before_script:
- - phantomjs --version
-script: 
+
+cache:
```

Comment:
https://github.com/travis-ci/travis-ci/issues/3225#issuecomment-200965782

Should cache phantomjs for us so that we don't always fail on installing